app/service/send_notification.py

```diff
@@ -78,3 +82,23 @@ def send_one_off_notification(service_id, post_data):
     )
 
     return {'id': str(notification.id)}
+
+
+def get_reply_to_text(notification_type, sender_id, service):
+    reply_to = None
+    if notification_type == EMAIL_TYPE:
+        if sender_id:
+            reply_to = dao_get_reply_to_by_id(service.id, sender_id).email_address
+        else:
+            service.get_default_reply_to_email_address()
+
+    elif notification_type == SMS_TYPE:
+        if sender_id:
+            reply_to = dao_get_service_sms_senders_by_id(service.id, sender_id).sms_sender
+        else:
+            reply_to = service.get_default_sms_sender()
+
+    elif notification_type == LETTER_TYPE:
```

Comment:
else raise an exception 'unknown type'?

Reply:
That validation happens in the check_template_is_for_notification_type, so it's not needed here.